projects/kubernetes/kubernetes/1-23/ATTRIBUTION.txt

```diff
@@ -2426,3 +2426,4 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ------
 
+
```

Comment:
nit: unnecessary new line?

Reply:
i want to force the kube project to build